### PR TITLE
Added JWTs/Cookies

### PR DESCRIPTION
**Pull request by jonslui**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -177,7 +179,16 @@ def login():
         user_to_login = user if user else user_by_email
         if bc.check_password_hash(user_to_login.password, in_password):
             login_user(user_to_login)
-            return user_schema.dump(user_to_login)
+            
+            # Creating JWT and Cookies
+            response = jsonify({
```
Comment (CNace13) [inline app/views.py:184]: Jon during registration we need to return the rio_key so users can enter it into the Rio client. I think its best if we return that along with the username

File: app/config.py [hunk 2]
```diff
@@ -8,4 +8,11 @@
 SQLALCHEMY_DATABASE_URI = DB_URI
 SECRET_KEY = 'S#perS3crEt_007'
 SQLALCHEMY_TRACK_MODIFICATIONS = False
-DEBUG = True
\ No newline at end of file
+DEBUG = True
+
+JWT_SECRET_KEY = 'S#perS3crEt_008'
```
Comment (CNace13) [inline app/config.py:13]: Wondering if we should use the same SECRET_KEY across the entire project. Genuinely unsure though

File: app/views.py (hunk 1 above)
Comment (jonslui) [reply app/views.py:184]: Sounds good, I'll add that in tomorrow

File: app/config.py (hunk 2 above)
Comment (CNace13) [reply app/config.py:13]: My initial thought was if either is revealed the whole thing falls apart anyway so why not just use the same key twice. But maybe the reverse is true, where we will be have secure if just one is compromised. For now this is fine IMO

**Pull request merged**